cmd/ddev/cmd/config.go

```diff
@@ -160,6 +160,11 @@ var ConfigCommand = &cobra.Command{
 			util.Failed("Could not write ddev config file: %v", err)
 		}
 
+		_, err = app.CreateSettingsFile()
+		if err != nil {
```

Comment:
Hrm. Maybe this isn't the correct spot to run this? Because it errors if the app type doesn't support settings. But we want to check that before running it.

Reply:
CreateSettingsFile() should just do nothing if there's no settings support. If that's not what's happening then something is wrong.

Reply:
Fixed to not throw an error if there is no support (ie: PHP apptype)